.github/workflows/ml-wheels.yml

```diff
@@ -0,0 +1,40 @@
+name: build-many-linux-wheels
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  ml-wheels:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+    
+      - name: upgrade pip
+        run: |
+          python -m pip install --upgrade pip
+
+      - name: Build manylinux Python wheels
+        uses: RalfG/python-wheels-manylinux-build@v0.4.2-manylinux2014_x86_64
+        with:
+          python-versions: 'cp37-cp37m'
```

Comment:
to speed up jobs. 
Will include other python versions once we successfully upload the package to Test PyPI

Reply:
excellent idea